docs/chapters/05-3.md

```diff
@@ -0,0 +1,278 @@
+{a_{m 1}} & {a_{m 2}} & {\cdots} & {a_{m n}}
+\end{array}\right]\left[\begin{array}{c}
+{x_{1}} \\
+{x_{2}} \\
+{\vdots} \\
+{x_{n}}
+\end{array}\right]
+$$
+
+There are two different equivalent representations of matrix-vector multiplication. The first one is 
+
+$$
+A\boldsymbol{x}=\left[\begin{array}{c}
+{\boldsymbol{a}^{(1)} \boldsymbol{x}} \\
+{\boldsymbol{a}^{(2)}\boldsymbol{x}} \\
+{\vdots} \\
+{\boldsymbol{a}^{(m)} \boldsymbol{x}}
+\end{array}\right]
+$$
+
+Where $\boldsymbol{a}^{(i)}$ is $i$th row vector of matrix $A$ and each $\boldsymbol{a}^{(i)}\boldsymbol{x}$ can be viewed as the **projection** from  input signal space to the kernel. 
+
+The other interpretation is 
+
+$$
+A\boldsymbol{x}=\left[\begin{array}{c}
+{a_{11}} \\
+{a_{21}} \\
+{\vdots} \\
+{a_{m 1}}
+\end{array}\right] x_{1}+\left[\begin{array}{c}
+{a_{12}} \\
+{a_{22}} \\
+{\vdots} \\
+{a_{m 2}}
+\end{array}\right] x_{2}+\ldots \ldots+\left[\begin{array}{c}
+{a_{1 n}} \\
+{a_{2 n}} \\
+{\vdots} \\
+{a_{m n}}
+\end{array}\right] x_{n}
+$$
+
+This can be viewed as weighted sum of the columns of matrix $A$.
+
+## Understanding 1D Convolution
+
+In this part we will discuss convolution, since we would like to explore the sparsity, stationarity, compositionality of the data.
+
+Instead of using the matrix $A$ above, we will change the matrix width to the kernel size $k$. Therefore, each row of the matrix is a kernel. We can use the kernels by stacking and shifting (see Fig 1). Then we can have $m$ layers of height $n-k+1$. 
+
+<center> 
+<img src="../05-3/Illustration_1D_Conv.png" alt="1" style="zoom:40%;" /><br>
+<b>Fig 1</b>: Illustration of 1D Convolution  
+</center>
+
+
+The output is $m$ (thickness) vectors of size $n-k+1$.
+
+<center> 
+<img src="../05-3/Result_1D_Conv.png" alt="2" style="zoom:40%;" /><br>
+<b>Fig 2</b>: Result of 1D Convolution  
+</center>
+
+
+Furthermore, a single input vector can viewed as a monophonic signal.
+
+<center>
+<img src="../05-3/Monophonic_Signal.png" alt="3" style="zoom:40%;" /><br>
+<b>Fig 3</b>: Monophonic Signal
+</center>
+
+
+Now, the input $x$ is a mapping
+
+$$
+x:\Omega\rightarrow\mathbb{R}^{c}
+$$
+
+where $\Omega=\{1,2,3,\dots\}\subset\mathbb{N}^1$ (since this is $1$ dimentional signal in a $1$ dimentional domain) and in this case channel number $c$ is $1$. And when $c=2$ this becomes a stereophonic system.
+
+For the $1D$ convolution, we can just compute the scalar product layer by layer. (see Fig 4)
+
+<center>
+<img src="../05-3/Layer_by_layer_scalar_product.png" alt="4" style="zoom:40%;" /><br>
+<b>Fig 4</b>: Layer-by-layer Scalar Product of 1D Convolution
+</center>
+
+
+## Dimension of Kernels and Output Width in PyTorch
+
+Tips: We can use ***question mark*** in IPython to get access to the documents of functions. For example,
+
+```python
+Init signature:
+nn.Conv1d(
+	in_channels,
+	out_channels,
+	kernel_size,
+	stride=1,
+	padding=0,
+	dilation=1,
+	groups=1,
+	bias=True,
+	padding_mode='zeros',
```

Comment:
Add `  # explanation` for each line. That is two spaces and a #, followed by a space and an explanation (without capital letter at the beginning).